### ISSUE TITLE
LanguageProperty not supported:

### ISSUE THREAD

**Issue by jason-fox** [ScorpioBroker/ScorpioBroker#416]
LanguageProperty is supported in NGSI-LD 1.6.1 - see: https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.6.jsonld

```console
curl -L 'http://localhost:9090/ngsi-ld/v1/entities/' \
-H 'Content-Type: application/json' \
-H 'Link: <http://context/ngsi-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
-d '{
    "id": "urn:ngsi-ld:City:001",
    "type": "City",
    "name": {
        "type": "LanguageProperty",
        "languageMap": {
            "el": "Κωνσταντινούπολις",
            "en": "Constantinople",
            "tr": "İstanbul"
        }
    }
}'
```

**Expected** - entity created

**Actual**  -  Error message.

```json
{
    "type": "https://uri.etsi.org/ngsi-ld/errors/InvalidRequest",
    "title": "Invalid request.",
    "detail": {
        "message": "There is an error in the provided json document"
    },
    "errorCode": 400
}
```

**Comment by ScorpioBroker** [comment]
Hi, 
no concise was unfortunately making Properties out of the languageMap values ... fixing it now

**Comment by ScorpioBroker** [comment]
Fixed in release 4.1.1

**Comment by jason-fox** [comment]
4.1.1 works for Normalized **LanguageProperty**, but not concise **LanguageProperty** - this fails with the same error.


```console
curl -L 'http://localhost:9090/ngsi-ld/v1/entities/' \
-H 'Content-Type: application/json' \
-H 'Link: <http://context/ngsi-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
-d '{
    "id": "urn:ngsi-ld:City:002",
    "type": "City",
    "temperature": {
        "value": 25,
        "unitCode": "CEL",
        "observedAt": "2022-06-30T00:00:00.000Z"
    },
    "address": {
        "value": {
            "streetAddress": "Viale di Valle Aurelia",
            "addressRegion": "Lazio",
            "addressLocality": "Roma",
            "postalCode": "00138"
        }
    },
    "location": {
        "type": "Point",
        "coordinates": [
            12.482,
            41.893
        ]
    },
    "population": {
        "value": 4342212,
        "observedAt": "2021-01-01T00:00:00.000Z"
    },
    "name": {
        "languageMap": {
            "el": "Ρώμη",
            "en": "Rome",
            "it": "Roma"
        }
    },
    "runBy": {
        "object": "urn:ngsi-ld:Adminstration:001"
    }
}'
```

**Comment by jason-fox** [comment]
This MERGE-PATCH also errors:

```console
curl -L -X PATCH 'http://localhost:9090/ngsi-ld/v1/entities/urn:ngsi-ld:City:002' \
-H 'Link: <http://context/ngsi-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
-H 'Content-Type: application/json' \
-d '{
    "humidity": 80,
    "name": {
        "languageMap": {
            "el": "Βερολίνο",
            "en": "Berlin",
            "it": "Berlino"
        }
    },
    "temperature": "urn:ngsi-ld:null"
}'
```

**Comment by jason-fox** [comment]
Fixed 4.1.15
